lesson1/program1.js

```diff
@@ -0,0 +1,6 @@
+const myfunction= (a,b)=>{
```

Comment:
myfunction tells us nothing about what the function does. you should use better names  like... `add`?